warema_wms/wms_controller.py

```diff
@@ -18,16 +18,17 @@
 RX_SHADE_STATE = '0431'
 
 TX_MOVE_SHADE = '0821'
-SHADE_POSITION = '03{}ffffff'
+SHADE_POSITION = '03{}dfffff'   # df = trial to move raffstores in tilt closed pos., working in console but not in HA
```

Comment:
```suggestion
SHADE_POSITION = '03{}ffffff'
```

You probably forgot to change this back after your trail.

Reply:
yes, sorry for that